src/utils/earn.utils.ts

```diff
@@ -48,3 +49,14 @@ export const calculateYouvesFarmingRewards = (
 
 export const aprToApy = (aprPercentage: number, compoundFrequency = APPROXIMATE_DAYS_IN_YEAR) =>
   ((1 + Number(aprPercentage) / 100 / compoundFrequency) ** compoundFrequency - 1) * 100;
+
+export const sortByNewest = (farmA: SingleFarmResponse, farmB: SingleFarmResponse) =>
+  new Date(farmB?.item?.firstActivityTime ?? Date.now()).getTime() -
+  new Date(farmA?.item?.firstActivityTime ?? Date.now()).getTime();
+
+export const sortByOldest = (farmA: SingleFarmResponse, farmB: SingleFarmResponse) =>
+  new Date(farmA?.item?.firstActivityTime ?? Date.now()).getTime() -
+  new Date(farmB?.item?.firstActivityTime ?? Date.now()).getTime();
```

Comment:
```suggestion
export const sortByOldest = (farmA: SingleFarmResponse, farmB: SingleFarmResponse) => -sortByNewest(farmA, farmB);
```
Please check for prettier errors before committing this suggestion